client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -112,6 +111,7 @@ const DesignSelector: FunctionComponent< Props > = ( { showPageSelector = false
 				className={ classnames( 'design-selector__grid-container', {
 					'is-page-selector-open': showPageSelector,
 				} ) }
+				tabIndex="-1"
```

Comment:
To fix typescript check failing:
```suggestion
				tabIndex={-1}
```

Reply:
Thank you @razvanpapadopol !